include/gz/sensors/Lidar.hh

```diff
@@ -244,6 +244,13 @@ namespace gz
       /// \return Visibility mask
       public: uint32_t VisibilityMask() const;
 
+      /// \brief Clamp a finite range value to min / max range.
+      /// +/-inf values will not be clamped because they mean lidar returns are
+      /// outside the detectable range.
+      /// NAN values will be clamped to max range.
+      /// \return Clamped range value.
+      private: double Clamp(double _range) const;
```

Comment:
why you don't use [std::clamp](https://en.cppreference.com/w/cpp/algorithm/clamp) ?

Reply:
needed to use special handling for clamping nan and inf values. [5c28cce](https://github.com/gazebosim/gz-sensors/pull/447/commits/5c28cce2df6f3a2a90ef857792acfe504eb47ec3)